docs/docusaurus.config.js

```diff
@@ -130,6 +130,12 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        appId: 'CYSR91W7SU',
+        apiKey: 'a5ab4af3642fdd7fbc07030593017797',
```

Comment:
Is there any danger to including these hard coded? Should they maybe be injected via ENV variables?

Reply:
They're all public keys so there's no security risk at all. They make that clear in the docs. If people took them and added them to their site all they would be doing is adding search for our docs on their site.

I took a look at other projects using this setup and they didn't bother to use env either.